plugin-bazel/src/main/kotlin/org/jetbrains/bazel/languages/bazelquery/elements/BazelqueryTokenType.kt

```diff
@@ -0,0 +1,10 @@
+package org.jetbrains.bazel.languages.bazelquery.elements
+
+import com.intellij.psi.tree.IElementType
+import org.jetbrains.bazel.languages.bazelquery.BazelqueryLanguage
+
+class BazelqueryTokenType(debugName: String) : IElementType(debugName, BazelqueryLanguage) {
+  override fun toString(): String = "Bazelquery:" + super.toString()
+
+  fun completionText(): String = super.toString().lowercase()
```

Comment:
will it cause some issues if tokens like `WHITE_SPACE`, `SQ_`, `DQ_` gets completed?